packages/form-js-viewer/src/render/hooks/useValuesAsync.js

```diff
@@ -35,11 +33,8 @@ export default function(field) {
   const [ valuesGetter, setValuesGetter ] = useState({ values: [], error: undefined, state: LOAD_STATES.LOADING });
   const initialData = useService('form')._getState().initialData;
 
-  const evaluatedValues = valuesExpression ? (useExpressionEvaluation(valuesExpression) || []) : [];
-
-  // re-calculate values if evaluated expression changes
-  const previousEvaluatedValues = usePrevious(evaluatedValues, [], [ evaluatedValues ]);
-  const evaluatedValuesChanged = !compare(previousEvaluatedValues, evaluatedValues);
+  const expressionEvaluation = useExpressionEvaluation(valuesExpression);
```

Comment:
I fixed our hook usage here, getting rid of the condition (hooks should not be conditionally evaluated, this confuses react/preact).